src/parseSerialize/schemas.ts

```diff
@@ -374,29 +369,51 @@ export const sponsorSchemaV1: TSchema = {
   ],
 }
 
-export const cancelSponsorSchemaV1: TSchema = {
+// ************************************************************************
+// This is a workaround for an issue with the Ledger app in the wallet ui
+// Once ledger app is fixed, the "Ledger" schemas will no longer be needed
+// see the issue: https://github.com/iicc1/ledger-app-lto/issues/3
+export const transferSchemaLedger: TSchema = {
   type: 'object',
   schema: [
     txFields.type,
     txFields.version,
-    txFields.chainId,
+    txFields.type,
     txFields.senderPublicKey,
-    txFields.recipient,
     txFields.timestamp,
+    txFields.amount,
+    txFields.fee,
+    txFields.recipient,
+    txFields.attachment,
+  ],
+}
+
+export const leaseSchemaLedger: TSchema = {
+  type: 'object',
+  schema: [
+    txFields.type,
+    txFields.version,
+    txFields.type,
+    txFields.senderPublicKey,
+    txFields.recipient,
+    txFields.amount,
     txFields.fee,
+    txFields.timestamp,
   ],
 }
+// ************************************************************************
 
 /**
  * Maps transaction types to schemas object. Schemas are written by keys. 0 - no version, n - version n
  */
 export const schemasByTypeMap = {
   [TRANSACTION_TYPE.GENESIS]: {},
   [TRANSACTION_TYPE.TRANSFER]: {
-    1: transferSchemaV1,
```

Comment:
Shouldn't we still have transferV1? So it is:
- `transferSchemaV1`
- `transferSchemaV2`
- `transferSchemaV3` (later on)
- `transferSchemaLedger`

Reply:
I renamed the schemas according to what Arnold mentioned on Telegram. Now we have `transferSchemaV1` and `transferSchemaV2`